### DISCUSSION TITLE
Manage Configuration file(*.yml) using submodule in git

### DISCUSSION THREAD

**Discussion by mangchhe** [withDogecoin/chaterview#14] [Ideas]
## Summary

We can manage *.yml files using submodule
This method can manage the configuration files without external sharing.

## How

- Create private github repo
- Add *.yml files to the private github repo
- When building the project, copy the *.yml files copy into `/src/main/resources`
  - configure `build.gradle.ks`
  - copy *.yml files before executing the build task

**Comment by BAEKJungHo** [comment]
## Command

- git submodule init
- git submodule update --remote chaterview-private
